sealing/garbage.go

```diff
@@ -125,10 +128,7 @@ func (m *Sealing) pledgeSector(ctx context.Context, sectorNum abi.SectorNumber,
 
 		// extend the slice with the newly-self-dealt piece
 		out = append(out, node.PieceWithDealInfo{
-			Piece: abi.PieceInfo{
-				Size:     ppi.Size.Padded(),
-				PieceCID: commcid.PieceCommitmentV1ToCID(ppi.CommP[:]),
-			},
+			Piece: ppi,
```

Comment:
No longer `PublicPieceInfo`, so the naming is a little confusing.

Reply:
roger that. updated.